src/parser/parser_support.c

```diff
@@ -9625,6 +9633,35 @@ pt_get_query_limit_value (PARSER_CONTEXT * parser, PT_NODE * query, DB_VALUE * l
   return NO_ERROR;
 }
 
+/*
+ * pt_get_query_limit () - get the limit value from a query
+ * return : error code or NO_ERROR
+ * parser (in)	      : parser context
+ * query (in)	      : query
+ */
+DB_BIGINT
+pt_get_query_limit (PARSER_CONTEXT * parser, PT_NODE * query)
```

Comment:
How about renaming to pt_get_query_limit_from_query?

Reply:
I fixed it.